Makefile

```diff
@@ -42,15 +42,20 @@ test:
 	@echo "===================================================================="
 	@RUSTFLAGS="-D warnings" REDISRS_SERVER_TYPE=unix RUST_BACKTRACE=1 cargo test --locked -p redis --all-features -- --test-threads=1 --skip test_cluster --skip test_async_cluster --skip test_module
 
+	@echo "===================================================================="
+	@echo "Testing async-std without TLS"
+	@echo "===================================================================="
+	@RUSTFLAGS="-D warnings" REDISRS_SERVER_TYPE=tcp RUST_BACKTRACE=1 cargo test --locked -p redis --features=async-std-comp,cluster-async,sentinel -- --nocapture --test-threads=1 --skip test_module
```

Comment:
just fyi, async-std did a release today I think after this CI ran https://github.com/async-rs/async-std/issues/1081